### ISSUE TITLE
cellFsRead generates WIN32 error 998/1784

### ISSUE THREAD

**Issue by tambry** [RPCS3/rpcs3#1466]
**Happens in following games:**
- Cloudberry Kingdom
- Steins;Gate 0
- CROSS CHANNEL ~ For all people ~ (998/1784)
- LittleBigPlanet™ Demo \[NPEA00147] (998/1784)
- The Guided Fate Paradox \[BLUS31312] (998)
- Hatsune Miku: Project DIVA F (998)
- JoJo's Bizarre Adventure: All-Star Battle (998)
- Naruto Shippuden: Ultimate Ninja Storm Generations (998)
- Naruto Shippuden: Ultimate Ninja Storm 3 (998)

**Comment by MyaniPT** [comment]
Also happens in BJLM61187 (CROSS CHANNEL ~ For all people ~)
Steins;Gate 0 also throws out some 0x6f8
```
{PPU Thread[0x24] (CRI FS File Access 2)[0x0018340c]} Exception: Unknown Win32 error: 0x6f8.
(in file ..\Utilities\File.cpp:690, in function fs::file::read)
Please report this to the developers.
```

**Comment by tambry** [comment]
That's quite a different error - ERROR_INVALID_USER_BUFFER.

**Comment by MyaniPT** [comment]
Well 0x6f8 happens less often than 0x3e6, preventing the game to boot at all like 0x3e6.
I got it to appear as the second game boot attempt this time
[rpcs3_474.zip](https://github.com/RPCS3/rpcs3/files/131285/rpcs3_474.zip)

Tl;dr: While trying to boot the game, sometimes it throws 0x3e6 and doesn't boot, sometimes it throws 0x6f8 and doesn't boot or it boots without any of those.

**Comment by tambry** [comment]
Ah, seems like it also occurs in cellFsRead so it's also probably relevant. Thank you.

**Comment by MyaniPT** [comment]
The log I sent you was for S;G 0 but the 0x6f8 also happens in Cross Channel so yeah

Log for Cross Channel @ [rpcs3_476.zip](https://github.com/RPCS3/rpcs3/files/133088/rpcs3_476.zip)

**Comment by AniLeo** [comment]
This hasn't happened to me yet in the latest build. Need further testing, only tried with S;G 0, but this issue may be gone (?)

**Comment by tambry** [comment]
Win32 error 998 (ERROR_NOACCESS) also occurs non-deterministically in LittleBigPlanet Demo.

**Comment by SakataGintokiYT** [comment]
@Nekotekina 
{PPU[0x98] Thread (CRI FS File Access 1) [0x00dc5b4c]} class std::runtime_error thrown: Win32 error: 998.
(in file C:\rpcs3\Utilities\File.cpp:793)

**Comment by SakataGintokiYT** [comment]
Do you know how to fix it ?

**Comment by tambry** [comment]
Not really. The documentation doesn't even specify a reason for why ERROR_NOACCESS (998) may occur.

**Comment by Nekotekina** [comment]
I know what causes it. And trying to fix it in fast, half-assed manner won't improve anything, so just wait. And it's irrelevant what games throw it, it may happen for anything.

**Comment by SakataGintokiYT** [comment]
*Now

(File.cpp:791 --> 834 )

{PPU[0x98] Thread (CRI FS File Access 1) [0x00dc5b4c]} class fmt::exception thrown: Win32 error: 998.
(in file C:\rpcs3\Utilities\File.cpp:834)

**Comment by AniLeo** [comment]
```
{PPU[0x34] Thread (CRI FS File Access 2) [0x00183410]} PPU: Function 'cellFsRead' aborted
{PPU[0x34] Thread (CRI FS File Access 2) [0x00183410]} class fmt::exception thrown: Win32 error: 1784.
(in file C:\rpcs3\Utilities\File.cpp:744)
```

Still an issue in the latest build (tested with Steins;Gate 0)
